Source/TextToSpeechV1/TextToSpeech.swift

```diff
@@ -736,6 +736,10 @@ public class TextToSpeech {
         // resources for WAV header format:
         // [1] http://unusedino.de/ec64/technical/formats/wav.html
         // [2] http://soundfile.sapp.org/doc/WaveFormat/
+
+        if data.count < 12 {
```

Comment:
Yes, 12 is magical and so is 4 and 8, which if you look at the existing code directly below this you will find those values hard coded. I am happy to use a #define or const to give it a meaningful name, if you like. Please advise if that is required.